azkustodata/kusto.go

```diff
@@ -200,12 +290,6 @@ func (c *Client) QueryToJson(ctx context.Context, db string, query Statement, op
 // Note that the server has a timeout of 10 minutes for a management call by default unless the context deadline is set.
 // There is a maximum of 1 hour.
 func (c *Client) Mgmt(ctx context.Context, db string, query Statement, options ...QueryOption) (*RowIterator, error) {
-	if stmt, ok := query.(Stmt); ok {
```

Comment:
Did this move somewhere?
Otherwise, why the change?

Reply:
not sure why, I'll revert - but this whole method is going to be removed soon anyway.